lib/modules/datasource/git-refs/base.ts

```diff
@@ -27,7 +28,16 @@ export abstract class GitDatasource extends Datasource {
   async getRawRefs({
     packageName,
   }: GetReleasesConfig): Promise<RawRefs[] | null> {
-    const git = simpleGit(simpleGitConfig());
+    const gitSubmoduleAuthEnvironmentVariables = getGitEnvironmentVariables([
+      this.id,
+    ]);
+    const gitEnv = {
+      // pass all existing env variables
+      ...process.env,
```

Comment:
Should this require `exposeAllEnv` to be set?

Reply:
I think that would make generally sense. 

I mimicked the current behavior, where per-default, all environment variables are passed to the downstream git process. 

If we restrict it, I think it would make sense to let  some variables through by default (like already existing `GIT_CONFIG_COUNT`, `GIT_CONFIG_KEY_n` and `GIT_CONFIG_VALUE_n`)

Reply:
Yeah, I'm not sure about it. Today we treat git as "trusted" while other package managers are untrusted. I guess let's keep it as-is - I'll resolve this comment